index.js

```diff
@@ -14,9 +14,9 @@
 //. [![CircleCI](https://img.shields.io/circleci/project/github/sanctuary-js/sanctuary/master.svg)](https://circleci.com/gh/sanctuary-js/sanctuary/tree/master)
 //. [![Gitter](https://img.shields.io/gitter/room/badges/shields.svg)](https://gitter.im/sanctuary-js/sanctuary)
 //.
-//. Sanctuary is a functional programming library inspired by Haskell and
-//. PureScript. It depends on and works nicely with [Ramda][]. Sanctuary
-//. makes it possible to write safe code without null checks.
+//. Sanctuary is a functional programming library inspired by Haskell
+//. and PureScript. It's stricter and more opinionated than [Ramda][].
+//. Sanctuary makes it possible to write safe code without null checks.
```

Comment:
I like it. I'll make the change you have proposed.

When Sanctuary first came into existence it was focused on avoiding the null checks which several Ramda functions require. Sanctuary's scope is now much broader.